app/main.py

```
@@ -230,7 +230,18 @@ async def health():
 
 
 def start_server(host: str = "0.0.0.0", port: int = 8766, local_dev: bool = False):
-    """Start the WebSocket server"""
+    """Start the Uvicorn server for the FastAPI application."""
+    # If the PORT environment variable is set, use it; otherwise, use the default.
+    try:
+        server_port = int(os.getenv("PORT", str(port)))
+    except ValueError:
+        logger.error(
+            f"Invalid value for PORT environment variable: {os.getenv("PORT")}. "
```

Pylint (3.3.7)

[error] Parsing failed: 'f-string: unmatched '(' (git.app.main, line 239)'

(E0001)